spec/services/pul_metadata_services/pulfa_record_spec.rb

```diff
@@ -8,20 +8,41 @@
   let(:source) { file_fixture("pulfa/#{pulfa_id}.xml").read }
 
   describe "#attributes" do
-    it "works" do
-      expected = {
-        title: ["19th Century Catalog and Correspondence, Pre-Vinton, 1811-"],
-        created: ["1865-01-01T00:00:00Z/1865-12-31T23:59:59Z"],
-        creator: ["Princeton University. Library. Dept. of Rare Books and Special Collections"],
-        publisher: ["Princeton University. Library. Dept. of Rare Books and Special Collections"],
-        memberOf: [{ title: "Princeton University Library Records", identifier: "AC123" }],
-        date_created: ["circa 1865"],
-        container: ["Box 1, Folder 2"],
-        extent: ["1 folder"],
-        heldBy: ["mudd"],
-        language: ["eng"]
-      }
-      expect(pulfa_record.attributes).to eq expected
+    context "with a component without orgination" do
+      it "works" do
+        expected = {
+          title: ["19th Century Catalog and Correspondence, Pre-Vinton, 1811-"],
+          created: ["1865-01-01T00:00:00Z/1865-12-31T23:59:59Z"],
+          creator: ["Princeton University. Library. Dept. of Rare Books and Special Collections"],
+          publisher: ["Princeton University. Library. Dept. of Rare Books and Special Collections"],
+          memberOf: [{ title: "Princeton University Library Records", identifier: "AC123" }],
+          date_created: ["circa 1865"],
+          container: ["Box 1, Folder 2"],
+          extent: ["1 folder"],
+          heldBy: ["mudd"],
+          language: ["eng"]
+        }
+        expect(pulfa_record.attributes).to eq expected
+      end
+    end
+
+    context "with a component that has origination" do
+      let(:pulfa_id) { "WC064_c1630" }
+      it "works" do
+        expected = {
+          title: ["Picnic west of Syracuse about 1885"],
+          created: ["1885-01-01T00:00:00Z/1885-12-31T23:59:59Z"],
+          creator: ["Savage, C. R. (Charles Roscoe),\n                1832-1909"],
```

Comment:
Should we find some way to parse this to not have all that white space and the newline? Maybe a different xpath query? Or however traject processes these strings?

Reply:
There's a lot of extra whitespace in these EADs; I was assuming this data was normalized somewhere further up the chain. Is there a general whitespace normalizer Figgy implements/uses, other than `strip`?